README.md

```diff
@@ -146,6 +146,12 @@ TIP: Use `tsc --watch` to update the build in a realtime while you develop the p
 
 TODO: we are planning to provide a template for TS projects. [task](https://www.pivotaltracker.com/n/projects/2452320).
 
+## Templates
+
+Using the `algob unbox-template` command, the developers can get a pre-built dApp project containing scripts to deploy assets and smart contracts with react.js interactive frontend. The templates heavily use [AlgoSigner](https://github.com/PureStake/algosigner) to securely sign and send transactions to an Algorand Blockchain Network. 
```

Comment:
maybe we can first explain what user will get and then how i.e `algob unbox.....`. just a thought

Reply:
@amityadav0  this doc is just an overview of templates and `algob unbox-template`, The real documentation can be found in the repo itself, i.e., `algo-builder-templates`. I have provided the link to it in the starting itself.

Reply:
what i meant here, our heading is `templates` so instead of starting with `algob unbox-templates` , we can first explain in a line what do we mean here and then the command. or this one is fine too.

Reply:
Okay. got it. I have made the apt changes.